exercises/book-store/description.md

```diff
@@ -43,8 +43,8 @@ This would give a total of:
 
 Resulting in:
 
-- 5 * (8 - 2.00) == 5 * 6.00 == $30.00
-- +3 * (8 - 0.80) == 3 * 7.20 == $21.60
+- 5 \* (8 - 2.00) == 5 \* 6.00 == $30.00
```

Comment:
We might code fence these so that they appear to be code (they are) and so that we do not need to escape the `*` for whatever reason they are being escaped here (formatting from markdown to html rendering, I suspect).

Reply:
The `*` get escaped so that a markdown renderer does not mistake them as emphasis syntax.

I had thought these lines are maths only, rather than code - to indicate the calculation that could be implemented by the student's code.

This quickly becomes apparent if we were to agree to format them as a code block then the next question would be to which language should that code block be set? The problem with that being that these files should be independent of any specific language.

Potentially, they could be set in a `text` code block, if that is agreeable?

Reply:
If they are math only, I do not think that `*` is the thing to do, instead use `×` which is the multiplication character.  If it is code, then use what is often used to multiply, which would be `*`.  Mathematics equality symbol is `=` rather than the programmatic `==`, I think.  So at this point we are kind of mixing math and programming in this one line.

So for me, this is now looking more "program expression" than "math expression".

Reply:
Since we want the dollar signs, or if we do, then we probably should make this more math.

Reply:
Updated in commit 0e8edaf.
Thanks for pasting the correct multiplication character, saved me looking it up 😄 .